index.html

```diff
@@ -6,10 +6,176 @@
+            <h3 class="text-white text-center pt-4">Main Program</h3>
+            <h3 class="text-danger text-center">____</h3>
+            <div class="mpcontents">
+                <div class="main-pogram-content justify-content-between" id="lecture">
+                    <a href=""><img class="" src="./image/lecture.png" alt="lecture logo"></a>
+                        <h5 class=" program-info text-danger font-weight-bold w-25">lecture</h5>
+                    <p class="program-details w-50 text-white m-0">Listen the speakers from various countries abouth the message of sharing and opening.</p>
+                </div>
+                <div class="main-pogram-content  justify-content-between" id="cc-exhibition">
+                    <a href=""><img class="" src="./image/cc_exhibition.png" alt="cc-exhibition logo"></a>
+                    <h5 class="program-info text-danger font-weight-bold w-25">CC Exhibition</h5>
+                    <p class="program-details w-50 text-white m-0">Appreciate various creating applying CC liscence of artists, Organize from art center nabi.</p>
+                </div>
+                <div class="main-pogram-content  justify-content-between" id="forum">
+                    <a href=""><img class="" src="./image/forum.png" alt="lecture logo"></a>
+                        <h5 class=" program-info text-danger font-weight-bold w-25">Forum</h5>
+                    <p class="program-details w-50 text-white m-0">Have the time to share your thoughts and opinions with experts for each topic.</p>
+                </div>
+                <div class="main-pogram-content  justify-content-between" id="workshop">
+                    <a href=""><img class="" src="./image/workshop.png" alt="lecture logo"></a>
+                        <h5 class=" program-info text-danger font-weight-bold w-25">Workshop</h5>
+                    <p class="program-details w-50 text-white m-0">Try to create your own creations using open source rather than just watching.</p>
+                </div>
+                <div class="main-pogram-content justify-content-between" id="cc-ignite">
+                    <a href=""><img class="" src="./image/program_icon.png" alt="lecture logo"></a>
+                        <h5 class=" program-info text-danger font-weight-bold w-25">CC Ignite</h5>
+                    <p class="program-details w-50 text-white m-0">Get opportunities to network with CC afilited around the world, also after summit.</p>
+                </div>
+            </div>
+            <a href="" id="see-the-pogram"><h5 class="see-the-pogram text-white text-center pt-4 fs-6">SEE THE WHOLE POGRAM</h5></a>
+            <div class="text-center pb-4" id="join-div">
+                <button id="join" class="join" type="button">Join CC Summit 2022</button>
+            </div>
+        </div>
+    </div>
+
+    <div class="speakers">
+        <h2 class="text-dark pt-4 text-center" id="speakers">"Featured Speakers"</h2>
```

Comment:
- [ ] Please ensure that your Featured speakers heading is bold and has the same font styling as the Behance design.
![image](https://user-images.githubusercontent.com/90556221/183044149-53566464-ebb4-4db8-a932-910339ef60bc.png)
![image](https://user-images.githubusercontent.com/90556221/183044469-51e00435-bf63-4f35-b428-607408c1673f.png)

Reply:
Changed the same as advised.